example/lib/shared/enemy/goblin.dart

```diff
@@ -105,15 +105,15 @@ class Goblin extends SimpleEnemy
   }
 
   @override
-  void receiveDamage(double damage, dynamic from) {
+  void receiveDamage(AttackFromEnum attacker, double damage, dynamic identify) {
```

Comment:
I think it would be nice to use named params, so it's easier to different 10 and 1 like the example in spikes

Reply:
I thought about this, but maybe if will we use named params stay maybe strange when you do override of this method.